src/common/config/decrypt.test.tsx

```diff
@@ -1,27 +1,40 @@
 import { getGSNRelayConfig } from "../../config";
 import sample from "../../test/fixtures/sample-config-ropsten.json";
+import awsSample from "../../test/fixtures/sample-config-ropsten-aws.json";
 import { ConfigFile } from "../../types";
-import { decryptWallet, getGsnRelaySigner } from "./decrypt";
+import { decryptWalletOrSigner, getGsnRelaySigner } from "./decrypt";
 
 const configFile = sample as ConfigFile;
+const awsConfigFile = awsSample as ConfigFile;
 const gsnConfig = getGSNRelayConfig("ropsten");
 
-describe("decryptWallet", () => {
+describe("decryptWalletOrSigner", () => {
   it("should return wallet when decryption is successful", async () => {
-    const wallet = await decryptWallet(configFile, "password", () => {});
-    expect(wallet.address).toBe("0x1245e5B64D785b25057f7438F715f4aA5D965733");
+    const wallet = await decryptWalletOrSigner(configFile, "password", () => {});
+    expect(await wallet.getAddress()).toBe("0x1245e5B64D785b25057f7438F715f4aA5D965733");
     expect(wallet.privateKey).toBe("0x416f14debf10172f04bef09f9b774480561ee3f05ee1a6f75df3c71ec0c60666");
   });
 
   it("should throw when decryption fails", async () => {
-    await expect(decryptWallet(configFile, "wrongPassword", () => {})).rejects.toThrow(/invalid password/);
+    await expect(decryptWalletOrSigner(configFile, "wrongPassword", () => {})).rejects.toThrow(/invalid password/);
+  });
+
+  it("should return signer when AWS decryption is successful", async () => {
+    const wallet = await decryptWalletOrSigner(awsConfigFile, "5TQto/r7ebS7rZRwavyi9qhxdWdDEoNVYTduFtEA", () => {});
```

Comment:
should the password be kept secret? or we can expose it

Reply:
should be able to expose it since it is in oa-cli test as well.

Reply:
kk